xml/System.ServiceModel/IExtensionCollection`1.xml

```diff
@@ -53,7 +53,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- <xref:System.ServiceModel.IExtensionCollection%601.Find%2A> returns an extension that implements (is or derives from) the specified type. If more than one extension implements the specified type, one is returned. The [!INCLUDE[indigo2](~/includes/indigo2-md.md)] implementation returns the most recently added, but this is not required of implementations. If no extension implements the specified type, <xref:System.ServiceModel.IExtensionCollection%601.Find%2A> returns null.  
+ <xref:System.ServiceModel.IExtensionCollection%601.Find%60%601%2A?displayProperty=fullName> returns an extension that implements (is or derives from) the specified type. If more than one extension implements the specified type, one is returned. The [!INCLUDE[indigo2](~/includes/indigo2-md.md)] implementation returns the most recently added, but this is not required of implementations. If no extension implements the specified type, <xref:System.ServiceModel.IExtensionCollection%601.Find%60%601%2A?displayProperty=fullName> returns null.  
```

Comment:
I saw the same behavior with https://docs.microsoft.com/en-us/dotnet/api/index?view=netframework-4.7&term=createordered.

https://docs.microsoft.com/en-us/dotnet/api/system.linq.iorderedenumerable-1.createorderedenumerable--1?view=netframework-4.7#System_Linq_IOrderedEnumerable_1_CreateOrderedEnumerable__1_System_Func__0___0__System_Collections_Generic_IComparer___0__System_Boolean_

vs.

https://docs.microsoft.com/en-us/dotnet/api/system.linq.iorderedenumerable-1.createorderedenumerable?view=netframework-4.7#System_Linq_IOrderedEnumerable_1_CreateOrderedEnumerable__1_System_Func__0___0__System_Collections_Generic_IComparer___0__System_Boolean_

I'll follow up tomorrow with @Sampy and @dend. Seems to be an API Browser issue. Link from within class page works.